runtimeconfig/README.rst

```diff
@@ -0,0 +1,47 @@
+Python Client for Google Cloud RuntimeConfig
+============================================
+
+    Python idiomatic client for `Google Cloud RuntimeConfig`_
+
+.. _Google Cloud RuntimeConfig: https://cloud.google.com/deployment-manager/runtime-configurator/
+
+-  `Homepage`_
+-  `API Documentation`_
+
+.. _Homepage: https://googlecloudplatform.github.io/google-cloud-python/
+.. _API Documentation: http://googlecloudplatform.github.io/google-cloud-python/
```

Comment:
Ah, yeah I see that now. They appear to be the same in a bunch of locations:

- https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/datastore/README.rst
- https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/vision/README.rst

I'll delete the "Homepage" link.